### ISSUE TITLE
[CAIA A11Y Research - PILOT]: Ask VA - Mobile Pilot - Form (August 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#89026]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled`
> - **PILOT DATE:** 8/1/2024 at 2 p.m. ET - [Zoom link to join](https://us06web.zoom.us/j/84716897287?pwd=Rqdcab1SSMlVlPxuM7bYUoAxLIJKSf.1)
> - **TYPES OF ASSISTIVE TECH:** Mobile `Device Type & AT Preferred`
>     - Info on AT being tested in user sessions needed, to prep and conduct pilot.
> - **PROTOTYPE:** [Staging Link](https://staging.va.gov/contact-us/ask-va-too/.)
> - **PILOT PREP NOTES:** [TEMPLATE (Duplicate) and link in task list](https://docs.google.com/document/d/1uZsaSNyaFThE7-sSkx-5hUXC8qMq2ZYu9ofxwlXh2SU/edit#heading=h.7b53y4dqihzk)

# CAIA A11y Support
This is a child ticket of the research ticket [87480](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87480) - created to track work, completed from this **MOBILE** pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @rileyorr 
- @sara-amanda 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Ask VA
- OCTO-DE product owner: Becky Phung
- Product manager: CeeCee O'Connor and Terri Hollar
- Designer: Tyler Gindraux, Thomas Michaud
- FE engineer: Eddie Otero, Hemesh Patel
- Product/team Slack channel: #ask-va-public

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** 8/1/2024 from 2- 3 p.m. ET
- **Prototype - Figma**: [Ask VA - Figma Prototype](https://www.figma.com/proto/nI0nYMdLamogjKq9t5atj8/Ask-VA-Prototype-R4-(Staging)?node-id=2001-81942&t=qOlK188ntSKlJfwi-1)
-  **Prototype - Staging:** [Ask VA - Staging Link](https://staging.va.gov/contact-us/ask-va-too/.) - `To be used in testing, per conversation guide.`
- **[Research Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Conversation%20guide.md)**
- **[Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md)**
</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

**12 Veterans:**
- minimum of 3 sessions with **screen reader** users
- minimum of 2 sessions with **alternative navigation** users
- minimum of 1 session with a **screen magnification** user

### Recruitment Criteria

<details><summary>Toggle to view recruitment criteria</summary>

- "We'd like to recruit people who use a mix of different screen readers, such as, [NVDA, JAWS, VoiceOver (Apple) and Talkback (Android)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenreaders)." 
    - **Source:** [Cohort 1: Screen Reader Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-1-screen-reader-users)
- "We'd like to recruit people who use a mix of [switch systems, Dragon, Voice Control (Apple), One-handed keyboards, a head mouse, or a sip and puff device](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Alternativenavigation)." 
    - **Source:** [Cohort 2: Alternative Navigation Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-2-alternative-navigation-users)
- "We'd like to recruit people who use a mix of different screen magnification software, such as, [ZoomText, ZoomText Fusion, Magnifier (Windows) and Zoom (Apple)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenmagnification)." 
    - **Source:** [Cohort 3: Screen Magnification Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-3-screen-magnification-users)

</details>

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** Riley
- [ ] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [ ] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Conversation%20guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#recruitment-criteria)
- [ ] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot - _**CAIA and the team will determine what AT will be used in this session, specifically.**_
- [ ] **Review prototype** in `Staging` - `Provided via convo guide`: [Staging Link](https://staging.va.gov/contact-us/ask-va-too/.)
- [ ] **Run pilot** with VFS Researcher
- [ ] **Optional: [Note Taking](https://docs.google.com/document/d/1kccoWW6S_6nAU3bBx0_x6ABGyu9AmIOBcgN2z-q3BhE/edit?usp=sharing):** If there is only one a11y, notes cannot be taken due to pilot participation. `[Pilot Notes Doc](LINK)`
```

**Comment by coforma-jamie** [comment]
### Findings

-   **Paragraphs**: Remove chunking of the paragraphs to allow the SR to read smoothly with the goal being not having to do multiple swipes to digest the content. Taking place in non-authenticated and authenticated views. TalkBack doesn't like nesting FYI. I imagine it's just because the controls quote unquote interrupt the text. VO is the same way.
-   **Logging in**: Authentication assumed to be necessary because "healthcare related."

-   **Page infinitely loading** - known error. Provide a heads up to the participant before this happens, if still taking place when testing begins.

-   **Zoom Bug:** Overlay meeting controls - keeping the screen visually crowded. Workaround: Facilitator paste in chat. Copy/paste of password worked, due to blocked typing.
-   **Number TalkBack Output**: 3,333 (reading out)
-   **Category**: first option is blank <option> - so that needs removed.
-   **Drop-down**: Not reading out the list count, similar to desktop.
-   **Old list of label names: **These have been fixed and won't show in real test.
-   **Focus**: Top of the page (not H1) for early (category + topic) pages.
-   **First few pages: **Similar work needed to Pilot 1 notes for desktop, as on mobile.
-   **Radio Buttons**: Address known issues with topics and category pages.
-   **Feedback**: Topic (H3) for the page you are on and then percentage, which is currently an H2: `<h2 class="vads-u-font-size--h6 vads-u-margin-top--1p5">18% complete with form</h2>`
-   **Demo prefill data:** helpful if available IE SSN to save time when testing.
-   **Font Size:** Differing font size on health facility page <labels> (when searching in Columbus, Ohio)
-   **Editing answers - alert: **Close button is not the first element in focus. Maria, a CAIA a11y who is visually impaired, says that this is the correct behavior, and that VA's modal component puts focus on the close button first, which is incorrect. She approved of [this guidance](https://accessibility.expedia.biz/pages/7-3).
-   **Editing answers - How should we contact you:** these are editable, and shouldn't be. And the "cancel" button isn't working.

You can also refer to our [Slack thread](https://dsva.slack.com/archives/C40B45NJK/p1722534638896059).
